### PR TITLE
Members list component

### PR DESCRIPTION
**Pull request by anguyen698**

### CODE REVIEW

**Review by hexalellogram — APPROVED** [review]
Looks good to me! Just waiting on one more person from frontend to review before we merge

**Review by jettbui — CHANGES_REQUESTED** [review]
styling seems a little off with displaying the member information, but JS looks good to me!

**Review by jettbui — APPROVED** [review]

**Pull request merged**
